server/resp/src/test/java/org/infinispan/server/resp/RespDecoderTest.java

```diff
@@ -45,8 +48,10 @@ static class Request {
 
    @BeforeClass
    public void beforeClass() {
+      RespServer server = mock();
```

Comment:
Is this related? I don't see any code using the time service now.

Reply:
I am unsure what is different, but running the tests in the terminal, testng started to run the access log test at the beginning. If the access log runs concurrently with the decoder, it has the trace enabled and causes an NPE on:

https://github.com/infinispan/infinispan/blob/9e94ababc8e23c8f270e7974bd2cc0141e804274/server/resp/src/main/java/org/infinispan/server/resp/RespHandler.java#L90

I could change the handler directly, though. Seems to make more sense.

Reply:
By unsure what is different I mean I don't know how testng decides what to run :(

Reply:
Aha, fun test leak issue. I would say we should really isolate the access logger tests themselves to prevent this type of thing, but I am fine with it this way.